nodebuilder/daser/opts.go

```diff
@@ -0,0 +1,15 @@
+package daser
+
+import (
+	"github.com/celestiaorg/celestia-node/das"
+	"go.uber.org/fx"
+)
+
+func withMetrics(enable bool) fx.Option {
```

Comment:
Can you export this function please?

Reply:
and you can remove the `enable` param and empty return, since it won't be in `daser/module.go` - this `WithMetrics` will only be called if metrics are enabled

Reply:
I wanted to pass bool param from config in this function. What is the benefit of making it exportable?